client/components/NumberOfRatings.jsx

```diff
@@ -1,25 +1,84 @@
 import React from 'react';
 import StarModal from './StarModal';
 
-const NumberOfRatings = (props) => (
-  <div>
-    <p>
-      <span>
-        <span onMouseLeave={props.onMouseLeave} onMouseEnter={props.onMouseOver} className="Star-Rating">
-          <span className="Stars-Span">
-            <i style={{ backgroundPosition: `-${props.starPosition}px -368px` }} className="Stars"></i>
-          </span>
-          <span className="A-Letter-Space"></span>
+class NumberOfRatings extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  render() {
+    const { onMouseLeave } = this.props;
+    const { onMouseOver } = this.props;
+    const { starPosition } = this.props;
+    const { numberOfRatings } = this.props;
+    const { starModalMouseEnter } = this.props;
+    const { starModalMouseLeave } = this.props;
+    const { starRating } = this.props;
+    const { stars } = this.props;
```

Comment:
You could reduce this all to one line... 

`const { onMouseLeave, onMouseOver, ..., stars } = this.props;`

Reply:
I realize this now. Not sure why at the time I didn't do this. Thank you for pointing this out to me.